### PR TITLE
[184] reduced linehieght for legend labels

### PR DESCRIPTION
**Pull request by JohnAGill**
Issue: https://github.com/project-vagabond/covid-green-app/issues/184

### CODE REVIEW

File: src/components/molecules/bar-chart.tsx [hunk 1]
```diff
@@ -277,6 +277,7 @@ const styles = StyleSheet.create({
   legendLabel: {
     ...text.small,
     textAlign: 'left',
-    marginLeft: 8
+    marginLeft: 8,
+    lineHeight: 15
```
Comment (davidjharding) [inline src/components/molecules/bar-chart.tsx]: Is there any reason that this lineHeight isn't coming in with the spread of `text.small`?

**Comment by ShaunBaker** [comment]
Can we link or reference the issue in description/commits?

**Comment by ShaunBaker** [comment]
@JohnAGill can you screenshot how this change actually fixes the issue?

https://github.com/project-vagabond/covid-green-app/issues/184

I would have thought the optimal fix would be to reduce the spacing to prevent wrapping...

**Comment by davidjharding** [comment]
Given that users can change their type size settings for a11y reasons, we do need to make sure this renders properly when it wraps as well.

**Comment by JohnAGill** [comment]
Screenshot 2020-09-04 at 10 18 20

Above is 1st gen SE on a dummy graph with the reduced horizontal padding

Screenshot 2020-09-04 at 10 19 18

Above is iphone 11 on a dummy graph with the reduced horizontal padding

I also tested on my XR and it looks the same as the above.

Let me know if you think the reduced space is too close we have a few px wiggle room

**Review by floridemai — APPROVED** [review]

**Pull request merged**